presto-docs/src/main/sphinx/plugin/redis-hbo-provider.rst

```diff
@@ -67,14 +70,14 @@ Steps
        <outputDirectory>plugin/redis-hbo-provider</outputDirectory>
     </fileSet>
 
-2. Add META-INF.services file in ``presto-hbo-provider/src/main/resources`` with the Plugin entry class com.facebook.presto.statistic.RedisProviderPlugin
```

Comment:
Plugins should not require a special build of Presto.  You should be able to build a plugin in isolation.  Can you update this documentation at a later stage?

Reply:
These instructions tell someone how to build all of Presto and make sure, as part of that build, that the Redis plugin ends up in the distributions `/plugin` directory.  But many people just drop JARs into their plugins directory, they don't build all of Presto.  I wonder if we can just leave instructions here telling people that they can do it that way.

Reply:
I see thanks for explaining, I have updated the docs!